Dockerfile

```diff
@@ -2,20 +2,22 @@ FROM ubuntu:16.04
 
 WORKDIR /dsw
 
+HEALTHCHECK --interval=5m --timeout=10s \
+  CMD curl -f http://localhost:3000/ || exit 1
+
 # Install necessary libraries
 RUN apt-get update && apt-get -qq -y install libmemcached-dev ca-certificates netbase wget gdebi-core curl
 RUN wget https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.trusty_amd64.deb && gdebi -n wkhtmltox_0.12.5-1.trusty_amd64.deb
-RUN wget https://github.com/jgm/pandoc/releases/download/2.2.1/pandoc-2.2.1-1-amd64.deb && gdebi -n pandoc-2.2.1-1-amd64.deb
+RUN wget https://github.com/jgm/pandoc/releases/download/2.7.2/pandoc-2.7.2-1-amd64.deb && gdebi -n pandoc-2.7.2-1-amd64.deb
 
 # Add built exectutable binary
 ADD .stack-work/install/x86_64-linux/lts-13.12/8.6.4/bin/dsw-server /dsw/dsw-server
 
-# Add configs
-ADD config/app-config.cfg.example /dsw/config/app-config.cfg
-ADD config/build-info.cfg /dsw/config/build-info.cfg
+# Add templates
 ADD templates /dsw/templates
 
-HEALTHCHECK --interval=5m --timeout=10s \
-  CMD curl -f http://localhost:3000/ || exit 1
+# Add configs
+ADD config/app-config.cfg /dsw/config/app-config.cfg
```

Comment:
Are you sure that you don't want to use `app-config.cfg.example`?

Reply:
It is weird to use `app-config.cfg.example` when you have `app-config.cfg` as required data file in `package.yml` (and not the example) and it is copied:

https://github.com/ds-wizard/dsw-server/blob/47411290b58f2dda945631ba798211259159256a/.travis.yml#L39

Do you think that the one with `example` should be in `Dockerfile`? Why?